pkg/testutils/docker/single_node_docker_test.go

```diff
@@ -93,7 +96,7 @@ func TestSingleNodeDocker(t *testing.T) {
 				},
 				volSetting: []string{
 					fmt.Sprintf("%s/testdata/single-node-test/docker-entrypoint-initdb.d/:/docker-entrypoint-initdb.d", pwd),
```

Comment:
Drive-by: It is also preferred to use `testutils.TestDataPath()` rather than literally referring to `testdata`. [#71928](https://github.com/cockroachdb/cockroach/issues/71928)